src/main/fc/init.c

```diff
@@ -310,11 +310,13 @@ void init(void)
     }
 #endif
 
+    systemClockSetHSEValue(systemConfig()->hseMhz * 1000000U);
+
 #ifdef USE_OVERCLOCK
     OverclockRebootIfNecessary(systemConfig()->cpu_overclock);
 #endif
 
-    delay(100);
+    delay(100); // XXX What is this for? Is it necessary?
```

Comment:
Apparently from multiwii: https://github.com/betaflight/betaflight/commit/d262c6e66c7862441c5a7e52248f4daf7ae12e2f#diff-ff21857f2bc5cf1319f56862a3aac350R111

Can I propose we remove it and forget it was ever there?

Reply:
Lol. I will.